ci/azure-pipelines.yml

```diff
@@ -129,47 +122,64 @@ stages:
       - job: snapshot_publish
         dependsOn: main
         condition: and(succeeded(),eq(variables['Build.Reason'], 'IndividualCI'))
+        variables:
+          PACKAGE_VERSION: $[ dependencies.main.outputs['BuildData.PACKAGE_VERSION'] ]
```

Comment:
we're not going to get extra spaces are in the version number   '$[ '  ?

Reply:
I copied it from the 1.4 branch so hopefully not! https://github.com/hyperledger/fabric-chaincode-java/blob/release-1.4/ci/azure-pipelines.yml#L122